### DISCUSSION TITLE
more then 100mb file error

### DISCUSSION THREAD

**Discussion by arslan2k** [community/community#129420] [Copilot Conversations]
### Select Topic Area

Bug

### Body

i am trying to push 139mb file to git hub through lfs but it is giving me large file error 
i need help!

**Comment by SUDEEP-M-SHETTY** [comment]
Verify LFS Tracking:
Confirm that the large file you're trying to push is actually tracked by Git LFS. You can check this by looking at the .gitattributes file in your repository.

Revert if Already Committed:
If the large file has been committed without LFS tracking, remove it from the repository history using git filter-repo or git filter-branch. This is essential because once a large file is committed without LFS, Git won't automatically migrate it to LFS.

Pushing Changes:
After ensuring the file is tracked by LFS and that the repository is set up correctly:

```bash
git add .
git commit -m "Add large file to LFS"
git push origin <branch_name>
```
Replace <branch_name> with the name of your branch.
